src/pages/Details.js

```diff
@@ -0,0 +1,72 @@
+import { useSelector } from 'react-redux';
+import Chart from '../components/Chart';
+import Header from '../components/Header';
+
+const CoinDetails = () => {
+  const { coinDetails } = useSelector((state) => state.coins);
+  const {
+    name,
+    symbol,
+    changePercent24Hr,
+    rank,
+    supply,
+    marketCapUsd,
+    volumeUsd24Hr,
+    priceUsd,
+    vwap24Hr,
+  } = coinDetails;
+
+  return (
+    <div className="h-screen md:w-1/2 lg:w-1/2 space-y-5 mx-auto p-2">
+      <Header />
+      <Chart
+        title1={name}
+        title2={symbol}
+        count={`${parseInt(changePercent24Hr, 10).toFixed(4)}%`}
+      />
+      <h3 className="px-2 py-5 text-gray-800 font-medium text-2xl">
+        Coin Description
+      </h3>
+      <ul className="font-medium text-lg">
+        <li className="flex justify-between px-2 py-5">
+          <span>Name :</span>
+          <span>{name}</span>
+        </li>
+        <li className="flex justify-between px-2 py-5 bg-teal-500">
+          <span>Symbol :</span>
+          <span>{symbol}</span>
+        </li>
+        <li className="flex justify-between px-2 py-5">
+          <span>Rank :</span>
+          <span>{rank}</span>
+        </li>
+        <li className="flex justify-between px-2 py-5 bg-teal-500">
+          <span>Supply :</span>
+          <span>{supply}</span>
+        </li>
+        <li className="flex justify-between px-2 py-5">
+          <span>Market Capital(USD) :</span>
+          <span>{marketCapUsd}</span>
+        </li>
+        <li className="flex justify-between px-2 py-5 bg-teal-500">
+          <span>Volume(USD) 24Hrs :</span>
+          <span>{volumeUsd24Hr}</span>
+        </li>
```

Comment:
- [ ] Some text elements of the details page are overflowing at the right side of the page, Please fix that to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/hamzaw9/metrics/assets/68796232/940f1064-7a94-41c4-9573-883c7b4e7408)  | ![image](https://github.com/hamzaw9/metrics/assets/68796232/d829870b-5285-43e0-869b-267e7c8b5728)  |